process/consensus.md

```diff
@@ -23,11 +23,12 @@ Polling consensus is done by the chair:
 2. The chair asks all participants to express their opinion to the question,
    asking in turn whether they are `Strongly For`, `For`, `Neutral`, `Against`,
    or `Strongly Against`. Participants vote for a single option by raising their
-   hand, or abstain entirely. Aggregate votes are recorded by the note-taker.
-3. If deemed relevant, the chair can ask certain participants if their wish to
+   hand, voting on a poll, or in chat for online meetings, or abstain entirely.
+   Aggregate votes are recorded by the note-taker.
+4. If deemed relevant, the chair can ask certain participants if their wish to
    explain their vote for the note-taker.
-4. The chair determines whether consensus was reached.
-5. A participant may decide to register a formal objection to the decision. An
+5. The chair determines whether consensus was reached.
+6. A participant may decide to register a formal objection to the decision. An
```

Comment:
Who is the director mentioned below? Is that the same as our W3C liaison?

Reply:
No, it's the director of the whole W3C. However the process for WG formal objections has changed (I think https://www.w3.org/2023/Process-20231103/#addressing-fo is the current process). Maybe this can be fixed in a separate change.

Reply:
Yeah, I'd prefer to defer reflecting W3C changes to a separate change, this is only for reflecting our current process in the document.